### PR TITLE
Add example using traits

### PR DESCRIPTION
**Pull request by LorenzoLeonardo**
Added an example using traits

### CODE REVIEW

**Review by blinking8888 — CHANGES_REQUESTED** [review]
Let's have the example in 1 file i.e. `examples/car.rs` and let's have the examples be shorter or more concise.

File: examples/with_traits/main.rs [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            }
+            CarKind::Suv(car) => {
+                let handle = atticus::run_actor(car, 1);
+                handle.requestor
+            }
+        }
+    }
+}
+
+async fn run_car(car: CarKind) {
```
[the remark this answers is not in the document]
Comment (blinking8888) [reply examples/with_traits/main.rs]: Let's move car.rs under `examples` and not anymore inside `examples/with_traits`

[the remark this answers is not in the document]
Comment (LorenzoLeonardo) [reply examples/with_traits/main.rs]: Done

[the remark this answers is not in the document]
Comment (blinking8888) [reply examples/with_traits/main.rs]: lgtm

**Review by blinking8888 — APPROVED** [review]
lgtm

**Pull request merged**